be/src/olap/storage_engine.cpp

```diff
@@ -185,6 +177,39 @@ OLAPStatus StorageEngine::_open() {
     return OLAP_SUCCESS;
 }
 
+OLAPStatus StorageEngine::_init_store_map() {
+    std::vector < std::pair<DataDir*> tmp_stores;
```

Comment:
???
```suggestion
    std::vector<DataDir*> tmp_stores;
```

Reply:
cherry-pick mistake :sweat:
When can we have a look at CI output? It will be enormously helpful.